lib/rules/indent.js

```diff
@@ -97,7 +97,7 @@ module.exports = {
                 const: DEFAULT_VARIABLE_INDENT
             },
             outerIIFEBody: null,
-            MemberExpression: 1
+            MemberExpression: null
```

Comment:
We like to keep options as all the same data type, so `null` isn't appropriate for this.